Assets/Scripts/Singletons/CurrentGameObjects.cs

```diff
@@ -81,6 +88,38 @@ public void addObjectsCollected(GameObject aPlant)
         return objectsCollected;
     }
 
+    // Return just potions created by the user
+    public List<Potion> GetPotionsCreated()
+    {
+        List<Potion> potions = new List<Potion>();
+
```

Comment:
So we create all of these potions regardless if I only have ingredients for one potion?

Reply:
this will only return the list of potions crafted once that list is being written to after the mini game is played

Reply:
Okay, maybe add a TODO here since this is hard coded right now to craft all these potions?